tools/spanner/purge_ttl.py

```diff
@@ -0,0 +1,24 @@
+# Purge Expired TTLs
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from google.cloud import spanner
+
+# Change these to match your install. 
+instance_id = 'spanner-test'
```

Comment:
Let's log an issue for a couple enhancements to this I can think of:

- it should delete from the batch table (I'm fine deferring this for now as I might be altering its schema shortly)
- these should probably be read from env variables. I'm tempted to say it should match syncstorage-rs's SYNC_DATABASE_URL format for consistency sake (though that will require url parsing code, the python client doesn't use that format)

Reply:
Meh, I'm ok with altering this PR to at least read the sync data from two env vars. 
I'm not quite sure about parsing the sync_database_url, a bit since I don't know if the cloud func can pull in the urllib. I may mess around a bit to see.

Reply:
Good point. These probably need a requirement file anyway, unless google-cloud-spanner is pre-installed on GCE nodes. write_batch.py may not necessarily run from there anyway.

I forgot to add this one to my orig. repo: https://github.com/pjenvey/spanner-tests/blob/master/requirements.txt

Reply:
looks like `urllib` is available for cloud functions. Added DSN parser. 

thanks! Good idea.